### DISCUSSION TITLE
ExplorerPatcher ENTRY in Task Manager?

### DISCUSSION THREAD

**Discussion by JazzyJazzjazzy** [valinet/ExplorerPatcher#889] [Q&A] [answered]
Hello,

Thank you for this GREAT software. May I ask what the RESOURCES REQUIRED are to run this software?

I can not find it in the TASK MANAGER to review it's footprint.

Respectfully,
Jay

**Comment by valinet** [comment, marked as answer]
It's not a standalone application. It's a module which is injected in various system applications. Consulting Task Manager is not that reliable of an indicator anyway, even if it were a standalone application.

What do you mean resources required? It depends on what features of the app you use, ofc. Basic stuff uses negligible memory: those are mainly hooks in the functions of the target application, that get called whenever you do certain stuff and that's it. Other features use some memory to do hold certain data structures, like Simple Window Switcher keeping a list of windows, the weather widget hosting an instance of WebView2. It depends... In terms of CPU usage, most of the things are callback based, they run once triggered and then the program waits to be signaled from the system, using no CPU.

I don't know what your expectations are.

**Reply by JazzyJazzjazzy** [reply]
Hello Sir,

I thank you for your kind explanations. 

I now understand how your software works. 

My expectation was to determine the performance penalty, if any, of running this software in the background.

Your explanation indicates that the impact would be minimal and dependent on what services I am using. 

The hooks and so forth would cause no slow down.

Thank you and much success to you, my friend.
